modules/test_user_dashboard/spec/services/update_user_spec.rb

```diff
@@ -25,7 +25,7 @@
       end
 
       it 'sets the test account to be checked out' do
-        expect(tud_account.checkout_time).to eq(@timestamp)
+        expect(tud_account.checkout_time.round(3)).to eq(@timestamp.round(3))
```

Comment:
Was there a reason we landed on round instead of truncate?

Reply:
To maintain consistency with the rest of the changes Trevor requested to the identity stuff. Before rounding, in linux/pipelines, one will have a precision of 9, and one will have a precision of 6 (truncated), so rounding to 3 should give consistent results without affecting the integrity of the spec (I believe).

Reply:
@ericboehs I adjusted slightly. Because the @timestamp is used in the process of updating the tud, I rounded it in in the definition of `@timestamp` instead. Note that `truncate` doesn't exist for time, only round, so that's our only option without taking a different approach